sprint-mission5/src/components/Alltem.jsx

```diff
@@ -0,0 +1,59 @@
+import Searchbox from "./Searchbox";
+import ItemSubmit from "./ItemSubmit";
+import DropDownbtn from "./DropDownbtn";
+import "./Alltem.css";
+import { getAllProducts } from "../api";
+import ListItem from "./ItemList";
+import { useEffect, useState } from "react";
+const Alltem = () => {
+  const [productList, setProductList] = useState([]);
+  const [order, setOrder] = useState("recent");
+  const handleRcentClick = () => {
+    setOrder("recent");
+    setIsOpen(false);
+  };
+  const handleFavoriteClick = () => {
+    setOrder("favorite");
+    setIsOpen(false);
+  };
```

Comment:
이거 같은 경우, 아래 하나로 해결 가능할 것 같아요.

```suggestion
 
  const handleOrderChange = useCallback((newOrder) => {
    setOrder(newOrder);
    setIsOpen(false);
  }, []);

```